src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -447,11 +449,7 @@ private static string GetFixCategory(DiagnosticSeverity severity)
                         }
 
                         if (!result.UpToDate)
```

Comment:
i'm still not super trustworthy of what is going on here.  in a world with pull diagnostics, i'm not sure what concept UpToDate means.  I hope we can remove this once we're entirely on pull diagnostics.